File: Exp 1.c
// Write a C program to find the search element in an array using Linear Search.

#include <stdio.h>
int main()
{
  int array[100], search, c, n;

 
  scanf("%d", &n);

 
  for (c = 0; c < n; c++)
    scanf("%d", &array[c]);

 
  scanf("%d", &search);

  for (c = 0; c < n; c++)
  {
    if (array[c] == search)
    {
      printf("%d \n", c+1);
      break;
    }
  }
  if (c == n)
    printf("%d isn't present in the array.\n", search);

  return 0;
}

File: Exp 1.py
# Write a Python program to find the largest and smallest number in an unsorted array.

array=[int(x) for x in input().split()]
n=len(array)
big=small=array[0]
for i in range(1,n):
    if (array[i]>big):
        big=array[i]
    if (array[i]<small):
        small=array[i]
print("Largest value =", big)
print("Smallest value =", small)


# Input: 10, 2, 3, 15, 8

# Output: Largest value = 15,     Smallest value = 2git 